wgsl/index.bs

```diff
@@ -3067,35 +3067,53 @@ Issue: Which index is used when it's out of bounds?
   <tr><td>*e1* : *T*<br> *e2* : *T*<br> *T* is *FloatVec*<td>`e1 % e2` : *T*<td>Component-wise floating point modulus (OpFMod)
 </table>
 
+
 <table class='data'>
-  <caption>Binary arithmetic expressions with mixed scalar, vector, and matrix operands</caption>
+  <caption>Binary arithmetic expressions</caption>
```

Comment:
The previous tables are also for binary expressions, of certain kinds.
I think the title should remain the same.